img/v/vancity.com.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 39.95 12.3"><path fill="#ef2e31" d="M5.35 1.08c.02.8-.16 4.69-1.67 6.58C2.16 5.76 1.98 1.88 2 1.08H.13C.11 2.37.37 6.98 3.06 9.2a.32.32 0 0 0 .22.1h.8a.32.32 0 0 0 .21-.1C7 6.98 7.24 2.37 7.22 1.08zm30.14 1.8h-3.54V1.07L30 1.61V2.9h-.84v1.2h.84v3.44c0 1.09.6 1.9 2.04 1.9.64 0 1.21-.14 1.53-.36l-.4-1.12c-.15.1-.4.15-.59.15-.33 0-.62-.11-.62-.7V4.08h1.72c.3 2.16 1.27 3.86 2.58 4.8a3.93 3.93 0 0 1-2.1 1.88l.55 1.4c3.21-1.33 4.86-5.65 5.12-9.28h-1.91c-.07 2.02-.4 3.49-.95 4.66-.88-.9-1.4-2.65-1.47-4.66M27.32.13c-.6 0-1.11.38-1.11.98 0 .59.51.97 1.1.97.6 0 1.11-.38 1.11-.97s-.51-.98-1.1-.98m-.98 9.19h1.95V2.89h-1.95zm0 0h1.95V2.89h-1.95zm-15.13-1.3a.2.2 0 0 1-.11.18c-.12.05-.32.08-.6.08-.76 0-1.12-.36-1.12-.91 0-.73.57-1.16 1.83-1.2zm-.79-5.25a6.85 6.85 0 0 0-2.55.48l.4 1.07c.48-.2 1.11-.4 1.76-.4.71 0 1.18.2 1.18.9v.27c-2.28.13-3.8.68-3.8 2.32 0 1.35 1.05 2.02 3.08 2.02.9 0 2.1-.18 2.52-.43.08-.05.15-.1.15-.29V4.99c0-1.3-.73-2.22-2.74-2.22m14.02 5.35c-.94 0-1.65-.73-1.65-1.98s.67-2.05 1.56-2.05c.25 0 .46.04.73.14l.43-1.16c-.41-.2-.86-.3-1.45-.3-1.92 0-3.23 1.3-3.23 3.43s1.38 3.24 3.27 3.24c.64 0 1.1-.1 1.5-.3l-.42-1.17a1.94 1.94 0 0 1-.74.15m-7.38-5.35a7.4 7.4 0 0 0-2.62.42c-.14.07-.21.12-.21.28v5.85h1.95v-5.2c0-.03.01-.04.08-.07.12-.04.42-.12.76-.12.8 0 1 .45 1 .92v4.47h1.95V5.04c0-.42-.02-.67-.09-.87-.26-.83-1.08-1.4-2.82-1.4"/></svg>
```

Comment:
Since the letters will be scaled down quite a bit, and the name _Vancity_ is displayed right next to it anyway, maybe we should just go with the "V" instead:
![](https://raw.githubusercontent.com/phallobst/phallobst/main/resterampe/vancity.com.svg)
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="41.1" height="41.1"><path d="M28.925 0c.1 4-.8 23.45-8.35 32.9-7.6-9.5-8.5-28.9-8.4-32.9h-9.35c-.1 6.45 1.2 29.5 14.65 40.6a1.6 1.6 0 0 0 1.1.5h4a1.6 1.6 0 0 0 1.05-.5C37.175 29.5 38.375 6.45 38.275 0z" fill="#ef2e31"/></svg>
```

Reply:
Thank you! I wanted that version of the icon, but couldn't find one in SVG format.